app/models/profile.rb

```diff
@@ -33,6 +33,7 @@ def decrypt_pii(user_access_key)
   end
 
   def recover_pii(personal_key)
+    personal_key = personal_key.tr('-', ' ')
```

Comment:
this probably needs more normalization than the simple `tr` if we can't trust the input as-is. I'll hunt a bit for the best way.